src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -851,6 +851,31 @@ def setCoresAndStreams(self, cores, nStreams, initialTask=None):
 
         return
 
+    def setGPUSettings(self, requiresGPU, gpuParams, initialTask=None):
+        """
+        Setter method for the workload GPU parameters.
+        It's responsible for setting whether GPUs are required or not; and
+        which GPU parameters to be used for that. This is done for every
+        task of this spec.
+        :param requiresGPU: string defining whether GPUs are needed. For TaskChains, it
+            could be a dictionary key'ed by the taskname.
+        :param gpuParams: GPU settings. A JSON encoded object, from either a None object
```

Comment:
What happens if it is a json encoded  from None. I can guess, of course, but it is a little bit obscure.

Reply:
This:
```
In [2]: json.dumps(None)
Out[2]: 'null'
```
which will be the default value in the specs/workflows. Also defined in the StdBase spec file.